docs/guide/cli.md

```diff
@@ -73,7 +73,6 @@ Run only [benchmark](https://vitest.dev/guide/features.html#benchmarking-experim
 | `--silent` | Silent console output from tests |
 | `--isolate` | Isolate environment for each test file (default: `true`) |
 | `--reporter <name>` | Select reporter: `default`, `verbose`, `dot`, `junit`, `json`, or a path to a custom reporter |
-| `--outputDiffLines <lines>` | Limit number of output diff lines up to `<lines>`. |
```

Comment:
Why was this option removed in a commit named chore(deps): update all non-major dependencies, and why was this option removal not mentioned in the [0.30.0](https://github.com/vitest-dev/vitest/releases/tag/v0.30.0) changelog?

I'm rather offended by such deceiving development practices.